dtfs-central-api/src/services/changeStream/changeStreamApi.ts

```diff
@@ -0,0 +1,46 @@
+import axios from 'axios';
+import { ChangeStreamDocument } from 'mongodb';
+
+/**
+ * Checks whether the document is a document that needs to be sent to the audit API
+ * and sends it if it is
+ * @param changeStreamDocument Change Stream Document event from mongodb API
+ * @returns
+ */
+export const postAuditDetails: (changeStreamDocument: ChangeStreamDocument) => Promise<void> = async (changeStreamDocument: ChangeStreamDocument) => {
+  try {
+    if (!process.env.AUDIT_API_URL || !process.env.AUDIT_API_USERNAME || !process.env.AUDIT_API_PASSWORD) {
+      console.error('Not updating audit API, audit API env var missing');
+      return;
```

Comment:
We could probably just throw a new `EnvironmentalVariablesMissingError` here and then let the error handling in `setupChangeStream` handle it -- what do you think?

Reply:
Didn't know this error existed! Will update :)